### ISSUE TITLE
Playback crashes after adding tracks to queue

### ISSUE THREAD

**Issue by FireSpirit15** [lavalink-devs/youtube-source#87]
### Basic Troubleshooting

- [x] I have checked for similar issues (open AND closed), and my issue is not a duplicate. (MANDATORY)
- [X] I have checked for pull requests that may already address my issue. (MANDATORY)
- [X] I am using (and my issue is reproducible on) the latest version of youtube-source.
- [X] My issue is reproducible with IPv6 rotation.

### Version of youtube-source

1.11.0

### The search query/queries, URL(s), playlist ID(s)/URL(s) or video ID(s)/URL(s) that triggered the issue

ytmsearch

### Code Example

N/A

### Exception and Stacktrace

Description: When playing, after adding one or more tracks to the queue, the playback crashes after about 1 minute with the following errors.
That issue seems to be present since the recent update to 1.11.0.

```
07:38:36.522 [lava-daemon-pool-playback-1-thread-2] WARN com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor -- Suspicious exception for playback of UM4n3jTu6Q4
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when decoding the track.
	at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:104)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
Caused by: java.lang.RuntimeException: Not success status code: 403
	at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.validateStatusCode(PersistentHttpStream.java:88)
	at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.attemptConnect(PersistentHttpStream.java:124)
	at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.connect(PersistentHttpStream.java:106)
	at dev.lavalink.youtube.track.YoutubePersistentHttpStream.internalRead(YoutubePersistentHttpStream.java:50)
	at dev.lavalink.youtube.track.YoutubePersistentHttpStream.internalRead(YoutubePersistentHttpStream.java:76)
	at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.read(PersistentHttpStream.java:194)
	at java.base/java.io.DataInputStream.readFully(DataInputStream.java:208)
	at com.sedmelluq.discord.lavaplayer.container.matroska.format.MutableMatroskaBlock.getNextFrameBuffer(MutableMatroskaBlock.java:54)
	at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.processFrameInBlock(MatroskaStreamingFile.java:390)
	at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.parseClusterSimpleBlock(MatroskaStreamingFile.java:354)
	at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.parseNextCluster(MatroskaStreamingFile.java:339)
	at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.provideFrames(MatroskaStreamingFile.java:315)
	at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.lambda$process$0(MatroskaAudioTrack.java:40)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.executeProcessingLoop(LocalAudioTrackExecutor.java:261)
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.executeProcessingLoop(LocalAudioTrackExecutor.java:234)
	at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.process(MatroskaAudioTrack.java:39)
	at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.processStatic(YoutubeAudioTrack.java:160)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:128)
	at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:74)
	... 5 common frames omitted
07:38:36.523 [lava-daemon-pool-playback-1-thread-2] DEBUG com.sedmelluq.discord.lavaplayer.player.AudioPlayer -- Firing an event with class TrackExceptionEvent
07:38:41.437 [JDA AudioConnection Guild: 919680333094481972 Sending Thread] DEBUG com.sedmelluq.discord.lavaplayer.player.AudioPlayer -- Firing an event with class TrackEndEvent
07:38:41.437 [JDA AudioConnection Guild: 919680333094481972 Sending Thread] DEBUG com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor -- Tried to stop track UM4n3jTu6Q4 which is not playing.
```

### What is your client configuration?

music

**Comment by Nansess** [comment]
![image](https://github.com/user-attachments/assets/85fefe8b-d441-481e-b3b8-0cd774f043e3)


dont use android client, plus 403 is a normal rate limit, use IP rotation and OAuth

**Comment by devoxin** [comment]
Possibly fixed in 1.11.1.
